setup.py

```diff
@@ -50,6 +50,7 @@
     "requests >=2.27.1",
     "scikit-learn >=1.2.2",
     "sqlalchemy >=1.4,<3.0dev",
+    "tabulate >= 0.9",
```

Comment:
plz justify the License of the new dependency.

Reply:
uses permissive MIT license.